portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,69 @@
+                <img src="../images/nyChurch.jpg" alt="Church Ceilings" width="500"/>
+                <img src="../images/musicHall.jpg" alt="Hanging Chandelier in a Lobby" width="500"/>
+            </div>
+        </div>
+        <div id="second">
+            <h3>My Cat</h3>
+            <div id="descrip2">
+                <p>Gracie is the family cat. She's seven years old and throughout those years she has never failed to
+                     be a great model for photographs. These are only a few of the many photos I have of her.</p>
+            </div>
+            <div id="photos2">
+                <img src="../images/catIntern.JPG" alt="Gracie with Intern Hat" width="500"/>
+                <img src="../images/catYawn.JPG" alt="Gracie Yawning" width="500"/>
+                <img src="../images/darkYawn.JPG" alt="Gracie Yawning" width="500"/>
+                <img src="../images/studyBuddy.JPG" alt="Gracie Sleeping" width="500"/>
+            </div>
+        </div>
+        <div id="third">
+            <h3>Drawing Inspiration</h3>
+            <div id="descrip3">
+                <p>These are photos that may not look like much, but when I took then I was trying to capture an element
+                     of the environment that I wanted to incorporate into a drawing. Recently I have had more time to practice drawing
+                     and so these photos serve have been serving as inspiration for what I draw.</p>
+            </div>
+            <div id="photos3">
+                <img src="../images/alcoveBench.jpg" alt="A Bench Hidden By Plants" width="500"/>
+                <img src="../images/cactusClose.PNG" alt="A Cactus" width="500"/>
+                <img src="../images/cactusTower.jpg" alt="A Cactus in a Tower" width="500"/>
+                <img src="../images/closerRPool.jpg" alt="A Closeup on a Fountain" width="500"/>
+                <img src="../images/curvePath.jpg" alt="A Pathway" width="500"/>
+                <img src="../images/dragonUp.PNG" alt="A Dragon" width="500"/>
+                <img src="../images/farDragon.JPG" alt="A Dragon from afar" width="500"/>
+                <img src="../images/fountainGlare.jpg" alt="A Fountain" width="500"/>
+                <img src="../images/grassClose.PNG" alt="Blades of Grass" width="500"/>
+                <img src="../images/ivyWall.jpg" alt="A Wall Covered in Plants" width="500"/>
+                <img src="../images/pavillion.PNG" alt="A Small Pavillion" width="500"/>
+                <img src="../images/plantsCurve.jpg" alt="A Curved Wall of Plants" width="500"/>
+                <img src="../images/rabbit.JPG" alt="A Rabbit" width="500"/>
+                <img src="../images/rabbitPool.jpg" alt="A Rabbit" width="500"/>
+                <img src="../images/stoneCactus.jpg" alt="A Cactus in a Stone Tower" width="500"/>
+                <img src="../images/walkway.jpg" alt="A Walkway" width="500"/>
+            </div>
+        </div>
```

Comment:
Visual inspection of these closing tags makes me think different amounts of indent are being used throughout the file. It'd be nice if the same amount of indent is used everywhere!

Reply:
You're right! Unfortunately every time Cloud Shell disconnects and has to be reloaded it changes the default tab size back to 4 instead of two and although I'm working on using the spacebar instead, I oftentimes still use tab out of habit.